pegasus/test/fixtures/deliverer/params/hoc_signup_2023_receipt_en.json

```diff
@@ -0,0 +1,3 @@
+{
+  "form_kind": "HocSignup2023"
+}
```

Comment:
super tiny nit: please end with a newline

Reply:
On it!